src/components/GeoMap.js

```diff
@@ -92,7 +92,6 @@ export default class GeoMap {
     this.layerDefs = this.svg.append('defs');
     this.layerDefs.html(`
     <filter id="blur"><feGaussianBlur stdDeviation="18" /></filter>
-    <filter id="blur-tz"><feGaussianBlur stdDeviation="2" /></filter>
```

Comment:
I just did the test and indeed this works better on Safari.
We should remove the filter for the time zone layer `.attr('filter', 'url(#blur-tz)')` on line 346

Reply:
@melalj  I've updated the code as you mentioned above. Please have a look.